### ISSUE TITLE
Pubspec doesn't declare compatibility with Dart 2

### ISSUE THREAD

**Issue by mit-mit** [loup-v/geolocation#40]
The Dart SDK constraints need to be updated to [declare compatibility with Dart 2 stable](https://www.dartlang.org/dart-2#upper-constraints-on-the-sdk-version).

It looks like this was fixed in https://github.com/loup-v/geolocation/commit/f32b900ec98f5251d30ebb189e0772863217863e#diff-fb2d52cd953afedcafc9fe7629c6f132, but never published?

**Comment by mit-mit** [comment]
cc @@lukaspili

**Comment by jarzac** [comment]
I have the same problem:

```
geolocation: ^0.2.1
```

```
flutter packages upgrade
Running "flutter packages upgrade" in app...
The current Dart SDK version is 2.1.0-dev.1.0.flutter-69fce633b7.

Because app depends on geolocation >=0.1.1 which requires SDK version <2.0.0, version solving failed.
pub upgrade failed (1)
Process finished with exit code 1
```

**Comment by prasant10050** [comment]
@jarzac - i have the same problem.

**Comment by mrdimaan** [comment]
same here

**Comment by nevi-me** [comment]
I tried loading the package from git, but thought the Dart 2.1 problem is solved, I'm unable to get locations from the library

**Comment by mit-mit** [comment]
Is doesn't look like this package is being maintained anymore. You may want to consider some of the other geolocation packages available on Pub:
https://pub.dartlang.org/flutter/packages?q=geolocation&api=0

**Comment by nevi-me** [comment]
Thanks @mit-mit, sucks a bit, but it's the nature of OSS and a relatively young ecosystem. I'm already maintaining a few private forks of libraries that aren't being updated anymore. I only develop for Android, so I can't do much on the iOS side of things.

**Comment by solid-software** [comment]
FYI - you can link to this repo directly (not through pub package) - what you can do is to write it that way:

  geolocation:
    git:
      url: https://github.com/loup-v/geolocation
      ref: f32b900

And it should work. Hope that maintainer will get back at some point. If you have any questions on that syntax - let me know.

**Comment by nevi-me** [comment]
I've moved to geolocator. I had previously linked to the url as you showed, but to master, I think using that ref would have worked.

**Comment by lukaspili** [comment]
Sorry for the delay. It should be fixed in the latest release 1.0.0.
